packages/common/src/ceramic-api.ts

```diff
@@ -71,12 +94,18 @@ export interface AdminApi {
    */
   getIndexedModels(): Promise<Array<StreamID>>
 
+  /**
+   * List indexed model streams with fields
+   */
+  getIndexedModelsWithFieldIndices(): Promise<Array<ModelFieldsIndex>>
```

Comment:
what if in the future there is extra information we want to return about models?  Then we'd need yet another method here.  What if we call this `getIndexedModelsInfo` or something like that, and it returns a `ModelInfo` which for now just has the streamid and the indices array, but could be extended with more data in the future?